parm/config/gfs/config.ocnanalprepobs

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Would prefer this config to be `config.prepoceanobs`
We currently have `config.prep`
Newer components would be `config.prep<component>obs`

Reply:
update elsewhere as necessary.